target/linux/imx6/image/lumi.mk

```diff
@@ -0,0 +1,33 @@
+#
+
+define Device/xiaomi_gateway-lumi
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Lumi Gateway
+  BOARD_NAME := xiaomi,gateway-lumi
```

Comment:
Is BOARD_NAME still required on this target?

Reply:
I'd like to use the same dtb for both Xiaomi dgnwg05lm and Aqara ZHWG11LM as they are identical.
BOARD_NAME here allows me to make it equal to dtb compatible section to allow sysupgrade.

Reply:
That sounds like a hack that only accidentally has the desired effect (BOARD_NAME is actually an ancient remnant fixing a specific upgrade issue). Please simply create one DTS per device (using shared DTSI files), and get rid of BOARD_NAME.

Reply:
Rewrote this part. Left `BOARD_NAME=xiaomi,dgnwg05lm` equal to the compatible in dts to support upgrade.
It is the only way to pass the check

https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/upgrade/nand.sh#L347-L353

Reply:
The trick is to use wildcards on the path, e.g. here in the generic version: https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/upgrade/nand.sh#L278

Getting rid of BOARD_NAME is highly desirable, it's an outdated sysupgrade mechanism and quite painful if one wants to change any names.

Reply:
I removed BOARD_NAME and fixed upgrade image checks